package.json

```diff
@@ -40,9 +40,9 @@
     "babel-polyfill": "^6.26.0",
     "bootstrap-vue": "^2.13.0",
     "core-js": "^3.4.3",
+    "direct-vuex": "^0.9.3",
```

Comment:
Why is this dependency introduced?

Reply:
She was already there but line 45, I think I move it inadvertently.